docs/main/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/openebs-on-kubernetes-platforms/gke.md

```diff
@@ -0,0 +1,259 @@
+ - Replicated PV Mayastor Installation on Google Kubernetes Engine
+ - Replicated PV Mayastor Installation on GKE
+ - Replicated PV Mayastor - Platform Support
+ - Platform Support
+ - Google Kubernetes Engine
+ - GKE
+description: This section explains about the Platform Support for Replicated PV Mayastor.
+---
+# Replicated PV Mayastor Installation on Google Kubernetes Engine
+
+This document provides instructions for installing Replicated PV Mayastor on Google Kubernetes Engine (GKE).
+
+## Prerequisites
+
+Before installing Replicated PV Mayastor, make sure that you meets the following requirements:
+
+- **Image**
+
+    Replicated PV Mayastor is supported exclusively by GKE clusters that are provisioned on the [Ubuntu node images](https://cloud.google.com/kubernetes-engine/docs/concepts/node-images) (ubuntu_containerd). It is necessary to specify the Ubuntu node image when you create the clusters.
+
+- **Hardware Requirements**
+
+    Your machine-type must meet the requirements defined in the [prerequisites](../rs-installation.md#prerequisites).
+
+- **GKE Nodes**
+
+    The minimum number of worker nodes that can be supported is three. The number of worker nodes on which IO engine pods are deployed should not be less than the desired replication factor when using the synchronous replication feature (N-way mirroring).
+
+- **Additional Disks**
+
+    Additional storage disks for nodes can be added as [local SSDs](https://cloud.google.com/kubernetes-engine/docs/concepts/local-ssd#block) during the cluster creation based on the machine types. These local SSDs should be created as a Block device storage by using the `--local-nvme-ssd-block` option and not as a ephemeral storage.
+
+    ```
+    gcloud container clusters create <CLUSTER_NAME> --machine-type <MACHINE_TYPE> --num-nodes <NUMBER_OF_NODES> --zone <ZONE> --local-nvme-ssd-block count=1 --image-type ubuntu_containerd
+    ```
+
+- **Enable Huge Pages**
+    
+    2MiB-sized Huge Pages must be supported and enabled on the storage nodes i.e. nodes where IO engine pods are deployed. A minimum number of 1024 such pages (i.e. 2GiB total) must be available exclusively to the IO engine pod on each node.
+    Secure Socket Shell (SSH) to the GKE worker node to enable huge pages. See [here](https://cloud.google.com/kubernetes-engine/distributed-cloud/vmware/docs/how-to/ssh-cluster-node) for more details.
+
+- **Kernel Modules**
+
+    SSH to the GKE worker nodes to load nvme_tcp modules.
+
+    ```
+    modprobe nvme_tcp
+    ```
+
+- **Preparing the Cluster**
+
+    See [here](../rs-installation.md#preparing-the-cluster) for instructions on preparing the cluster.
+
+- **ETCD and LOKI Storage Class**
+
+    GKE storage class - standard (rwo) should be used for ETCD and LOKI.
+
+## Install Replicated PV Mayastor on GKE
+
+To install Replicated PV Mayastor using Helm, refer to the [installation steps](../../../../quickstart-guide/installation.md#installation-via-helm) in the Quickstart Guide.
```

Comment:
The requirement of GKE SC for etcd and loki should be mentioned under the installation, preferably with the exact helm install command having --set options.
cc @datacore-tilangovan @niladrih

Reply:
Sure, will update the command.